### PR TITLE
Feature lpdev 247 payment requests

### PR DESCRIPTION
**Pull request by olegtrofimenko**

### CODE REVIEW

File: src/Lykke.Service.PayBackoffice/Areas/LykkePay/Controllers/AssetsController.cs [hunk 1]
```diff
@@ -103,20 +103,20 @@ public IActionResult Index()
...
         {
-            var assets = await _payInternalClient.GetAvailableAsync(merchant);
+            var assets = await _payInternalClient.GetPersonalAvailableAssetsAsync(merchant);
             var vm = new AddAssetsByMerchantDialogViewModel
             {
                 Caption = "Add assets to merchant",
                 MerchantId = merchant,
-                AssetsPayment = assets != null ? assets.AssetsPayment : string.Empty,
-                AssetsSettlement = assets != null ? assets.AssetsSettlement : string.Empty,
+                PaymentAssets = assets != null ? assets.PaymentAssets : string.Empty,
```
Comment (tarurar) [inline src/Lykke.Service.PayBackoffice/Areas/LykkePay/Controllers/AssetsController.cs]: `asset?.PaymentAssets ?? string.Empty` would be more readable

File: src/Lykke.Service.PayBackoffice/Areas/LykkePay/Controllers/AssetsController.cs [hunk 2]
```diff
@@ -103,20 +103,20 @@ public IActionResult Index()
...
+                SettlementAssets = assets != null ? assets.SettlementAssets : string.Empty,
             };
             return View(vm);
         }
         [HttpPost]
         public async Task<ActionResult> AddAssetByMerchant(AddAssetsByMerchantDialogViewModel vm)
         {
-            await _payInternalClient.SetAvailabilityByMerchantAsync(new UpdateAssetAvailabilityByMerchantRequest()
+            await _payInternalClient.SetPersonalAvailableAssetsAsync(new UpdateAssetAvailabilityByMerchantRequest()
```
Comment (tarurar) [inline src/Lykke.Service.PayBackoffice/Areas/LykkePay/Controllers/AssetsController.cs]: Why not use any mapper? For example, AutoMapper.

File: src/Lykke.Service.PayBackoffice/Helpers/PaymentRequestStatusHelper.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        /// <summary>
+        /// The payment request was successfully refunded.
+        /// </summary>
+        Refunded,
+
+        /// <summary>
+        /// An error occurred during processing payment request.
+        /// </summary>
+        Error,
+        Cancelled
```
Comment (tarurar) [inline src/Lykke.Service.PayBackoffice/Helpers/PaymentRequestStatusHelper.cs]: Please add description for Cancelled

**Review by tarurar — APPROVED** [review]

File: src/Lykke.Service.PayBackoffice/Areas/LykkePay/Views/PaymentRequests/PaymentRequestsPage.cshtml [hunk 4]
```diff
@@ -38,6 +45,15 @@
...
+            $statusSelect.select2().on("change",
+                function(e) {
+                    requests.doRequest({
+                        url: '@Url.Action("PaymentRequestsList")',
+                        formId: '#frmPaymentRequests',
+                        divResult: '#paymentrequestsList'
+                    });
+                });
             $('.select2-container--default').width(200);
             requests.doRequest({ url: '@Url.Action("PaymentRequestsList")', formId: '#frmPaymentRequests', divResult: '#paymentrequestsList' });
```
Comment (vitaliidasaev) [inline src/Lykke.Service.PayBackoffice/Areas/LykkePay/Views/PaymentRequests/PaymentRequestsPage.cshtml:58]: this default request and `select2().on("change"` can be in one function, e.g. `getPayments()`

**Pull request merged**